profiles/models.py

```diff
@@ -0,0 +1,48 @@
+"""
+Models for user profile
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+
+
+class Profile(models.Model):
+    """
+    A user profile
+    """
+    PRIVATE = 'private'
+    PUBLIC_TO_MM = 'public_to_mm'
+    PUBLIC = 'public'
+    ACCOUNT_PRIVACY_CHOICES = (
+        (PRIVATE, 'Private'),
+        (PUBLIC_TO_MM, 'Public to logged in users'),
+        (PUBLIC, 'Public to everyone'),
+    )
+
+    user = models.ForeignKey(User)
+    account_privacy = models.TextField(
```

Comment:
I didn't read the StackOverflow answer to mean that they are identical in Postgres, just that there was no performance difference.  For Django there are fewer functional differences than if we weren't using an ORM.